### PR TITLE
Update Spanish translation of email template

### PR DESCRIPTION
**Pull request by rostow**
Hi there! I have updated the Spanish translation of the email template as per issue #511 . I hope you find it useful.

### CODE REVIEW

**Comment by maneesha** [comment]
Thanks for this @rostow !

My Spanish is not nearly good enough to review this so a review from one other person would be great.  Maybe @raynamharris  @orchid00 @dpshelio

File: topic_folders/instructor_training/email_templates_trainers.md [hunk 1]
```diff
@@ -105,27 +105,36 @@ Best/Saludos,
 [ sender name ]
 
 
-##### Spanish: Trainee Did Not Pass Teaching Demo
+##### Spanish: Trainee Invited to Repeat Teaching Demo
```
Comment (orchid00) [inline topic_folders/instructor_training/email_templates_trainers.md]: traducir

File: topic_folders/instructor_training/email_templates_trainers.md [hunk 2]
```diff
@@ -105,27 +105,36 @@ Best/Saludos,
...
+
+No dudes en ponerte en contacto si tienes cualquier pregunta!
 
 Saludos,
 
 [nombre del remitente]
 
+
+
 ##### Spanish: Trainee Did Pass Teaching Demo
```
Comment (orchid00) [inline topic_folders/instructor_training/email_templates_trainers.md]: traducir

**Comment by orchid00** [comment]
I've reviewed and approved the text. 
2 titles need to be translated, which can be done after merging. I have no rights to merge.

**Comment by orchid00** [comment]
Gracias @rostow Pedro, muy buena

**Comment by rostow** [comment]
Anytime! Let me know if you need more support with your project :)

**Comment by maneesha** [comment]
Gracia @rostow y @orchid00 !

**Pull request merged**